Explorer/Assets/Scripts/ECS/Unity/Visibility/Systems/VisibilitySystem.cs

```diff
@@ -0,0 +1,31 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using DCL.ECSComponents;
+using ECS.Abstract;
+using ECS.Unity.PrimitiveRenderer.Components;
+
+namespace ECS.Unity.Visibility.Systems
+{
+    [UpdateInGroup(typeof(SimulationSystemGroup))]
+    public partial class VisibilitySystem : BaseUnityLoopSystem
```

Comment:
I think it should be executed after `ComponentInstantiationGroup`, otherwise there is a possibility to delay visibility resolution by one frame